service/controller/v12/key/key.go

```diff
@@ -15,6 +15,7 @@ const (
 	clusterTagName      = "GiantSwarmCluster"
 	installationTagName = "GiantSwarmInstallation"
 	organizationTagName = "GiantSwarmOrganization"
+	VmssDeploymentName  = "cluster-vmss-template"
```

Comment:
Not used?

Reply:
Used 3 times. 
create.go lines 59 and 119 and 704